test/views/declaration/DeclarantExporterViewSpec.scala

```diff
@@ -84,32 +85,65 @@ class DeclarantExporterViewSpec extends PageWithButtonsSpec with Injector {
       checkAllSaveButtonsAreDisplayed(createView())
     }
 
-    onJourney(STANDARD, SIMPLIFIED, OCCASIONAL) { implicit request =>
-      "display 'Back' button that links to 'Link DUCR to a MUCR' page" in {
-        val backButton = createView().getElementById("back-link")
+    "display links" when {
 
-        backButton must containMessage(backToPreviousQuestionCaption)
-        backButton must haveHref(LinkDucrToMucrController.displayPage.url)
+      onJourney(STANDARD, SIMPLIFIED, OCCASIONAL) { implicit request =>
+        "display 'Back' button that links to 'Link DUCR to a MUCR' page" in {
+          val backButton = createView().getElementById("back-link")
+
+          backButton must containMessage(backToPreviousQuestionCaption)
+          backButton must haveHref(LinkDucrToMucrController.displayPage.url)
+        }
       }
-    }
 
-    onSupplementary { implicit request =>
-      "display 'Back' button that links to 'Are you the exporter' page" in {
-        val backButton = createView().getElementById("back-link")
+      onSupplementary { implicit request =>
+        "display 'Back' button that links to 'Are you the exporter' page" in {
+          val backButton = createView().getElementById("back-link")
 
-        backButton must containMessage(backToPreviousQuestionCaption)
-        backButton must haveHref(ConsignmentReferencesController.displayPage.url)
+          backButton must containMessage(backToPreviousQuestionCaption)
+          backButton must haveHref(ConsignmentReferencesController.displayPage.url)
+        }
+      }
+
+      onClearance { implicit request =>
+        "display 'Back' button that links to 'Declarant Details' page" in {
+          val backButton = createView().getElementById("back-link")
+
+          backButton must containMessage(backToPreviousQuestionCaption)
+          backButton must haveHref(DeclarantDetailsController.displayPage.url)
+        }
       }
     }
 
-    onClearance { implicit request =>
-      "display 'Back' button that links to 'Declarant Details' page" in {
-        val backButton = createView().getElementById("back-link")
+    "hide links" when {
```

Comment:
Need to specify that this is testing the situation where the draft dec is an amendment draft